MAINTAINERS.yml

```
@@ -2809,6 +2809,36 @@ Toolchain Integration:
   labels:
     - "area: Toolchains"
 
+Toolchain ARC MWDT:
```

It would be nice to follow the existing convention:

```suggestion
"Toolchain: ARC MWDT":
```

For example, we have `"Drivers: ADC":` for drivers.

---

Done, although it seems the file could benefit from an overhaul, as there are several places not following that practice.

While updating for new entries, then I updated the existing `Toolchain Integration` --> `"Toolchain Integration"` at same time.